### PR TITLE
Modified course icons for smaller screens

### PR DESCRIPTION
**Pull request by heyitsmichellelin**
## Summary
Turned Course Info Button from class component to function component so the icon can disappear with smaller screen sizes. This corresponds to issue #260.

## Test Plan
Run the program on local host and resize the window to make sure the button functions properly.

### CODE REVIEW

File: src/components/SectionTable/CourseInfoButton.js [hunk 1]
```diff
@@ -1,26 +1,45 @@
...
+//                onClick={(event) => {
+//                    window.open(redirectLink);
+//                }}
+//            >
+//                {text}
+//            </Button>
+//        );
+//    }
+//}
+
```
Comment (ChaseC99) [inline src/components/SectionTable/CourseInfoButton.js]: We can delete this now :) 
```suggestion

```

File: src/components/SectionTable/CourseInfoButton.js [hunk 2]
```diff
@@ -1,26 +1,45 @@
...
-    }
+function CourseInfoButton({ text, icon, redirectLink }) {
+    const isMobileScreen = useMediaQuery('max-width: 750px)');
+
+    return (
+        <Button
+            startIcon={!isMobileScreen && icon}
+            variant="contained"
+            size="small"
+            style={{ marginRight: '4px', backgroundColor: '#385EB1', color: '#fff' }}
```
Comment (ChaseC99) [inline src/components/SectionTable/CourseInfoButton.js]: While you're working on this, could you make this style a separate class? You can look at other files to see how we do styles

File: src/components/SectionTable/CourseInfoButton.js [hunk 3]
```diff
@@ -1,26 +1,45 @@
...
+//            >
+//                {text}
+//            </Button>
+//        );
+//    }
+//}
+
 CourseInfoButton.propTypes = {
     text: PropTypes.string.isRequired,
     icon: PropTypes.object.isRequired,
```
Comment (ChaseC99) [inline src/components/SectionTable/CourseInfoButton.js]: I don't think these prototypes work right now. See this link for how to do it with function components: https://reactjs.org/docs/typechecking-with-proptypes.html#function-components

**Review by ChaseC99 — CHANGES_REQUESTED** [review]
This is looking great! Just have a few suggestions and then it should be good to go.

File: src/components/SectionTable/CourseInfoButton.js (hunk 2 above)
Comment (heyitsmichellelin) [reply src/components/SectionTable/CourseInfoButton.js]: I tried to make style into a class but it keeps giving me this error in the console and a blank white page after I do a search.

Comment (ChaseC99) [reply src/components/SectionTable/CourseInfoButton.js]: We can try to debug more later, but for now try this:

- Add `classes` as prop. 
function CourseInfoButton ( { <code>classes</code>, text, icon, redirectLink }) {

- Delete line 17.
```diff
- const { classes } = { text, icon, redirectLink };
```

**Review by ChaseC99 — APPROVED** [review]
Looks great!

**Pull request merged**